README.md

```diff
@@ -256,13 +256,13 @@ Cypress .visit() function is locked to visiting a single domain per test. This b
 Here is an example command:
 
 ```
-CYPRESS_APP_ENV=test CYPRESS_UK=true npm run cypress:interactive
+CYPRESS_APP_ENV=test CYPRESS_UK=true npm run cypress
```

Comment:
Can we also mention the cypress smoke variable?

Reply:
Done :)